spec/lib/synapse/file_output_spec.rb

```diff
@@ -1,8 +1,9 @@
 require 'spec_helper'
 require 'fileutils'
+require 'synapse/config_generator/file_output'
```

Comment:
I agree, but right now in synapse we don't group specs yet. I think punt and followup.